docs/cri/registry.md

```diff
@@ -190,6 +190,51 @@ DEBU[0001] PullImageResponse: &PullImageResponse{ImageRef:sha256:78096d0a5478896
 Image is up to date for sha256:78096d0a54788961ca68393e5f8038704b97d8af374249dc5c8faec1b8045e42
 ```
 
+
+### Configure Registry Credentials Example - Dockerhub (docker.io)
+
+Confirm your dockerhub account works as expected
+pushing an image to it as follows:
+
+```console
+docker login -u <username> -p <password> 
+
+docker pull polarpoint/jb:0.1.0
```

Comment:
What's this image? https://hub.docker.com/r/polarpoint/jb

Reply:
Hi @AkihiroSuda  it's just an image I use for jsonnet bundling, The purpose of my PR is tp provide an example of how to get around the docker hub rate limiting by providing credentials to cri/containerd.

Reply:
Better to use `hello-world` perhaps?

Reply:
Or just `<user>/<image>:<tag>`